knowledge-base/common-change-selected-localization-keys.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Localize selected localization keys with custom strings
+description: How to modify selected localized strings without maintaining the complete version of the localization resource files
+type: how-to
+page_title: Localize selected localization keys with custom strings
+slug: common-kb-localize-selected-localization-keys
+position:
+tags: telerik, blazor, localization, key, resources, files, partial, string
+ticketid: 1601981, 1607465, 1604787
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Telerik UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+If I want to change a single localized string, is there any mechanism to fall back to component integrated localization? 
+
+Is there a way to not maintain a complete version of your localization files across all Telerik versions only because I want to change a single string? 
+
+How to achieve partial localization? 
+
+If I use a limited part of a component, I want to avoid providing and maintaining a full set of localized resources.
+
+## Solution
+
+To localize selected keys with custom strings:
+
+1. As a prerequisite, create a project with Telerik UI for Blazor.
+2. Modify the `Program.cs` file like this:
```

Comment:
The reader must dig to see what exactly they should add or change. Better replace **most** of the existing code with `// ...`, so that it's easier to spot the required changes. For example, see step 5.4 here - https://docs.telerik.com/blazor-ui/getting-started/server-blazor